Userland/Libraries/LibWeb/Layout/BlockFormattingContext.cpp

```diff
@@ -414,9 +414,6 @@ CSSPixels BlockFormattingContext::compute_auto_height_for_block_level_element(Bo
             auto const& child_box_state = m_state.get(*child_box);
 
             // Ignore anonymous block containers with no lines. These don't count as in-flow block boxes.
-            if (child_box->is_anonymous() && child_box->is_block_container() && child_box_state.line_boxes.is_empty())
-                continue;
```

Comment:
If we are removing the logic, we should also remove the comment explaining the logic :)